ush/ioda/bufr2ioda/bufr2ioda_acft_profiles_prepbufr.py

```diff
@@ -157,7 +161,11 @@ def bufr_to_ioda(config, logger):
 
     logger.debug(f"Executing QuerySet to get ResultSet ...")
     with bufr.File(DATA_PATH) as f:
-        r = f.execute(q)
+        try:
+            r = f.execute(q)
+        except Exception as err:
+            logger.info(f'Return with {err}')
+            return
```

Comment:
Are these fatal exceptions?

Reply:
Good question.  There can be cycles for which a given dump file does not contain expected bufr subsets.  What do we want to happen?  If it is critical data a _fatal_ abort may be appropriate.  A _warning_ may suffice for non-critical data.  _info_ could be used for data-of-opportunity.   

My preference is to let the DA team working on bufr2ioda converters discuss and decide how to handle the `except` case.  The agreed upon approach should be consistently applied across all converters.

Reply:
Sounds good to me.  I simply made a note as part of the review.

Reply:
Both "expectation" and "warning" make the program return to the main program and exit with error or warning messages, respectively.  

There are two situations to give error or warning messages: 
case-1.  missing input data file
case-2. input file exists, but some or all of the required subsets are missing.  

The first one is easy --- give exception and error messages in the log to raise flags to check

The second one is a bit more complicated. 
For some data types, such as radiances, only one subset is in the BUFR file. If the subset is missing, it will be caught as missing input data. 
However, for data types like SATWIND AMV, using GOES as an example, there are multiple subsets (  "subsets"          : [ "NC005030", "NC005031", "NC005032", "NC005034", "NC005039" ]) in the BUFR file. So, we may run into a situation where only some subsets are included (some are missing). In this situation, we want to continue the program to process the non-missing subsets.  

**@PraveenKumar-NOAA and @nicholasesposito 
Please check and test your converters for various data scenarios and come up with a strategy to handle them properly.  I will check radiance, ozone, and satwind.**  

The ObsProc re-engineering team may need some more time to investigate possible data scenarios and how to handle them properly.  Later, as @dtkleist suggested, we will select a period (1-2 months) to do a pressure test for the converters.  

To avoid delaying merging this PR, I suggest that Russ's fixes for the converter issue with missing files or subsets be merged in as the first cut of improvement for the converters.